tools/getkeys/package-lock.json

```diff
@@ -1,18 +1,4632 @@
 {
   "name": "@fluid-internal/getkeys",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I think you may be on a more recent version of node/npm than the repo is currently configured for. We should still be generating v1 lock files.

Reply:
Fixed

Reply:
@chentong7 this file still reports `lockFileVersion=2`